packages/auth/package.json

```
@@ -21,10 +23,17 @@
     "firebase",
     "auth"
   ],
+  "dependencies": {
+    "@expo/config-plugins": "^4.0.18"
```

```suggestion
    "@expo/config-plugins": "^4.1.4"
```
Other packages have this version as dependency